### PR TITLE
Add experiment_notes column to import/export file formats

### PR DESCRIPTION
**Pull request by annehaley**

### CODE REVIEW

File: miqa/core/tasks.py [hunk 1]
```diff
@@ -327,27 +333,6 @@ def perform_export(project_id: Optional[str]):
...
-                        )
-                    artifacts = last_decision.user_identified_artifacts.items()
-                    row_data += [
-                        last_decision.decision,
-                        last_decision.creator.email,
-                        last_decision.note.replace(',', ';'),
-                        last_decision.created,
-                        ';'.join([artifact for artifact, value in artifacts if value == 1]),
-                        location,
-                    ]
```
Comment (dchiquito) [inline miqa/core/tasks.py:350]: Was this just accidentally duplicated?

**Review by dchiquito — COMMENTED** [review]
I don't see any changes to the import/export test. Is this tested implicitly somehow?

File: miqa/core/tasks.py (hunk 1 above)
Comment (annehaley) [reply miqa/core/tasks.py:350]: yes, I think as the result of a merge.

**Comment by annehaley** [comment]
> I don't see any changes to the import/export test. Is this tested implicitly somehow?

The experiment_notes column is optional, so it didn't affect the test we have, but I can add another test that uses all the optional columns.

EDIT: The new test revealed errors that I fixed, so it was helpful! :)

**Pull request merged**